stdlib/public/core/Integers.swift.gyb

```diff
@@ -1779,6 +1882,164 @@ ${operatorComment(x.nonMaskingOperator, False)}
   }
 % end
 
+  @_transparent
+  func quotientAndRemainder(
+    dividingBy divisor: Self
+  ) -> (quotient: Self, remainder: Self) {
+    return quotientAndRemainder(dividingBy: divisor, roundingQuotient: .down)
```

Comment:
If I'm reading correctly, this is a change from the current behavior, which acts as if `.towardZero` is passed. Ditto with `divided(by:)` and `remainder(dividingBy:)` below. Is that intentional?

Reply:
Semi-intentional. I'm still trying to decide if this will be part of the actual proposal.